### PR TITLE
Default Avatar selection.

### PR DESCRIPTION
**Pull request by IvanSpirandelli**
# Default Avatar Selection

**The process is working in general. To test it out:**
* Click on account
* Click avatar of your choice
* Reload

**Stuff that is Left to do:**
* The buttons for selecting the avatars should look nicer.
* Is there a way to update the message part of the page/ the shown avatars without reloading?
* Where do we want to save the information of what is a default avatar? DB? Right now it is just a list passed from the endpoint "/change_avatar/"

Still pretty happy with my progress.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -134,6 +134,15 @@ async def send_message(outgoing: OutgoingMessage, current_user: UserPydantic = D
     message_request = MessageRequest(sender_id=sender_id, recipient_id=recipient.id, content=outgoing.content)
     create_message(message_request, db)
 
+@app.post("/change_avatar/")
```
Comment (amueller-89) [inline main.py]: should be of type "get"

File: templates/SPA-loginbar.html [hunk 2]
```diff
@@ -29,6 +29,7 @@
     </div>
 </div>
 
+
```
Comment (amueller-89) [inline templates/SPA-loginbar.html]: naja ok

File: templates/SPA-loginbar.html [hunk 3]
```diff
@@ -48,9 +49,9 @@
 </div>
 
 <div id="modal-invalidpw" class="ui mini modal">
-    <div class="header">Login failed</div>
```
Comment (amueller-89) [inline templates/SPA-loginbar.html:51]: unintended changes??? please fix

File: templates/SPA-loginbar.html [hunk 4]
```diff
@@ -99,11 +100,70 @@
...
+        </div>
+    </div>
+    <div class="actions">
+        <div class="ui black deny button">
+            ok
+        </div>
+    </div>
+</div>
+
+    <style>
```
Comment (amueller-89) [inline templates/SPA-loginbar.html]: collect stylings somewhere nice in future refactoring?

File: templates/SPA-loginbar.html [hunk 5]
```diff
@@ -99,11 +100,70 @@
...
+
 <script>
 
-    // TODO avatar waehlen zB
-    function account(){
-        console.log("account settings i guess")
+    function account() {
+        const response = $.ajax({
+            url: "/change_avatar/",
+            type: "POST",
```
Comment (amueller-89) [inline templates/SPA-loginbar.html]: GET!!! brudi

**Review by amueller-89 — CHANGES_REQUESTED** [review]
da guckste aber nochma drueber wa.
pff also leute schickt das arbeitsamt...

**Review by IvanSpirandelli — COMMENTED** [review]
I like it

**Pull request merged**